src/Cake.Common/Tools/NuGet/Pack/NuGetPacker.cs

```diff
@@ -175,6 +176,18 @@ private ProcessArgumentBuilder GetArguments(FilePath filePath, NuGetPackSettings
                 builder.Append(settings.Verbosity.Value.ToString().ToLowerInvariant());
             }
 
+            // Properties
+            if (settings.Properties != null && settings.Properties.Count > 0)
+            {
+                if (settings.Properties.Values.Any(x => x == null))
```

Comment:
@daveaglick  Now, when @devlead said `""` he clearly meant string.Empty.  He must not have had his "StyleCop Head" on :wink: